internal/pkg/cli/pipeline_init.go

```diff
@@ -771,6 +771,7 @@ func buildPipelineInitCmd() *cobra.Command {
 		Example: `
   Create a pipeline for the services in your workspace.
   /code $ copilot pipeline init \
+  /code  --name my-pipeline
   /code  --url https://github.com/gitHubUserName/myFrontendApp.git \
```

Comment:
what do you think of this to match our suggestions?
```suggestion
  /code  --name my-frontend-app-main \
  /code  --url https://github.com/gitHubUserName/my-frontend-app.git \
```

Reply:
will also update examples for `status` and `show`